devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -303,78 +303,257 @@ bool WholeBodyDynamicsDevice::openEstimator(os::Searchable& config)
     return true;
 }
 
-bool WholeBodyDynamicsDevice::openDefaultContactFrames(os::Searchable& config)
+bool WholeBodyDynamicsDevice::openContactFrames(os::Searchable& config)
 {
     yarp::os::Property prop;
     prop.fromString(config.toString().c_str());
 
-    yarp::os::Bottle *propContactFrames=prop.find("defaultContactFrames").asList();
-    if(propContactFrames==0)
-    {
-       yError() <<"wholeBodyDynamics : Error parsing parameters: \"defaultContactFrames\" should be followed by a list\n";
-       return false;
-    }
+    yarp::os::Bottle *propDefaultContactFrames=prop.find("defaultContactFrames").asList();
+    yarp::os::Bottle *propOverrideContactFrames=prop.find("overrideContactFrames").asList();
+    yarp::os::Bottle *propContactWrenchType=prop.find("contactWrenchType").asList();
+    yarp::os::Bottle *propContactWrenchDirection=prop.find("contactWrenchDirection").asList();
+    yarp::os::Bottle *propContactWrenchPosition=prop.find("contactWrenchPosition").asList();
 
-    defaultContactFrames.resize(propContactFrames->size());
-    for(int ax=0; ax < propContactFrames->size(); ax++)
+    if(propOverrideContactFrames==0)
     {
-        defaultContactFrames[ax] = propContactFrames->get(ax).asString().c_str();
+        //overrideContactFrames don't exist -> considering defaultContactFrames
+        yInfo() << "wholeBodyDynamics : openContactFrames :  Parsing the parameters from defaultContactFrames.\n";
```

Comment:
This message may be useful for you while developing the feature, but as you may imagine if we print something for every parameter that we parse, the output of the device would became quite confusing.

Reply:
I understand, we may remove this one and keep the one related to `overrideContactFrames`. What do you think?

Reply:
I think we should keep consistency with the existing codebase, so do not print anything (as we do for all the other options). However this is a matter of taste, so it is not a big problem even if  the print remains.

Reply:
Okay. Let's keep the consistency. I removed them both in https://github.com/robotology/whole-body-estimators/pull/48/commits/a91ed5422a088256a65c7c27d838184f94cb28ba